spr/spr.go

```diff
@@ -187,7 +188,7 @@ func (sd *stackediff) UpdatePullRequests(ctx context.Context) {
 //  pull request. This one merge in effect merges all the commits in the stack.
 //  We than close all the pull requests which are below the merged request, as
 //  their commits have already been merged.
-func (sd *stackediff) MergePullRequests(ctx context.Context, upto *int) {
+func (sd *stackediff) MergePullRequests(ctx context.Context, upto *int, mergeMethod githubv4.PullRequestMergeMethod) {
```

Comment:
you don't actually need to pass in mergeMethod as an argument because you have access to the config struct here `sd.config`
you should be able to get the value and handle the error here instead of the main routine.